### ISSUE TITLE
PLEASE RUN THIS THING

### ISSUE THREAD

**Issue by Cucuzacu** [radmanplays/Eaglercraft-1.21#1]
run it, it's perfectly safe

**Comment by OrangeMan432** [comment]
I ran it, it works as intended. I don't see what issue your having with it.

**Comment by Zmonify** [comment]
> I ran it, it works as intended. I don't see what issue your having with it.

no way, this dude is not lying? run this on your chromebook, it DOSNT overload your system until it force restarts itself.

**Comment by Cucuzacu** [comment]
yeah i somehow managed to stop it before it was too late with task manager, idk why I did that. its NOT some sort of ur an idiot but 2024 version

**Comment by OrangeMan432** [comment]
> > I ran it, it works as intended. I don't see what issue your having with it.
> 
> Ain't no way this dude is lying? Do not run this on your chromebook, it overloads your system until it force restarts itself.

I'm not lying, it does what it is intended to do. "Cooking your chromebook 🔥🔥🗣🗣🗣 (literally(check the code))"

**Comment by Cucuzacu** [comment]
lol now i understood
u were telling the truth 💀

**Comment by radmanplays** [comment]
trololololol

**Comment by caplayz** [comment]
idk what you are tripping about it runs perfectly 60fps

**Comment by N00BK4NG** [comment]
nah lmao bro thought

**Comment by deep-v2** [comment]
LMAO

**Comment by milesarndt-cloud** [comment]
How does it work

**Comment by 87218-svg** [comment]
yo I have the real 1.20 I feel bad for whoever fell for it anyways here it is

[wasm-gc 1.20.html](https://github.com/user-attachments/files/24971716/wasm-gc.1.20.html)

its an html cuz I forgot the website link lol
